en/simulation/jmavsim.md

```diff
@@ -121,6 +121,13 @@ This allows a faster testing cycle (restarting jMAVSim takes significantly more
 
 JMAVSim can be used for multi-vehicle simulation: [Multi-Vehicle Sim with JMAVSim](../simulation/multi_vehicle_jmavsim.md).
 
+## Headless Mode
```

Comment:
You've put it under the wrong heading - needs to go above `Multi-Vehicle Simulation` and the level needs to change from `##` to `###`.

FWIW This is excellent. Should allow me to run JMAVSim in Ubuntu in a VM, which currently fails due to graphics driver issues.

Reply:
Fixed and force-pushed.